opendbc/car/honda/carstate.py

```diff
@@ -257,6 +269,21 @@ def update(self, cp, cp_cam, _, cp_body, __) -> structs.CarState:
     if self.CP.carFingerprint in HONDA_BOSCH_RADARLESS:
       self.lkas_hud = cp_cam.vl["LKAS_HUD"]
 
+    # # Low speed steer alert logic; for cars with steer cut off above 6 m/s
```

Comment:
commented out for the moment. need a way to get the state of the lateral controls